### PR TITLE
Move publishing to GH

### PR DESCRIPTION
**Pull request by ruiaraujo**
Travis publishing is broken.

Content copied from https://docs.github.com/en/actions/guides/publishing-nodejs-packages#publishing-packages-using-yarn

### CODE REVIEW

**Comment by ruiaraujo** [comment]
:+1:

**Comment by ponimas** [comment]
:+1:

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    # Setup .npmrc file to publish to npm
+    - uses: actions/setup-node@v2
+      with:
+        node-version: 14
+        registry-url: 'https://registry.npmjs.org'
+    - run: yarn install
```
Comment (boopathi) [inline .github/workflows/publish.yml:15]: a build step is required I suppose

Comment (ruiaraujo) [reply .github/workflows/publish.yml:15]: No, it is part of the publish workflow.

**Pull request merged**